.github/workflows/pmm2-testsuite.yml

```diff
@@ -127,7 +131,39 @@ jobs:
           sleep 30
 
       - name: Setup PMM2-Client
-        run: sudo -E bash -x ./pmm-tests/pmm2-client-setup.sh --pmm_server_ip 127.0.0.1 --client_version ${{ env.CLIENT_VERSION }} --admin_password admin --use_metrics_mode no
+        run: |
```

Comment:
could we just have a separate GitHub reusable action that takes client_version as input parameter and sets up the client on GitHub worker?

Reply:
yes, we can, but how many? we have right now 4 install forks install "dev-latest", "pmm2-rc" and "pmm2-latest". And 2 setup forks: package setup and tarball setup. 
Also please note that this one is not final setup refactoring - it's the step intermediate state before moving it into "qa-integration"

Reply:
updated with 2 setup steps